DatadogRUM/Sources/RUMVitals/VitalInfoSampler.swift

```diff
@@ -41,12 +41,22 @@ internal final class VitalInfoSampler {
 
     private var timer: Timer?
 
+    private static var maximumFramesPerSecond: Double {
+        get {
+#if os(visionOS)
+            return 120.0
```

Comment:
Would be great to add a comment explaining that it's hardcoded until visionOS is publicly available

Reply:
Sorry I didn't get around to doing this, I can create another PR with a comment if you wish.